test-suite/misc/universes/build_all_stdlib.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+echo "Require $(find ../../../theories ../../../plugins -type f -name "*.v" | \
+        sed 's/^.*\/theories\///' | sed 's/^.*\/plugins\///' | sed 's/\.v$//' | sed 's/\//./g') ."
```

Comment:
I'm surprised that you don't make the `Makefile` call this script as well like you usually do.

Reply:
What do you mean? This is just a shell version of what's in makefile.vofiles

Reply:
Yes indeed, but usually you would take out small pieces of Makefile, put them in shell scripts and have both the Makefile and dune call the script.

Reply:
It is more complicated than that for this case, if you look at the way the make rules work. Anyways this should be a promoted file and thus checked in the CI I think.

Reply:
I don't get why things are done like that in Makefile.build, and consequently why you are duplicating the logic here.
One wants the target "test-suite" to depend on the "stdlib" one. What is the point of generating a file that loads the entire stdlib? Rather than duplicating this logic, you should aim at the most natural way of expressing dependencies among targets. Do you have a good reason for doing things this way in dune?

Reply:
Indeed @gares, I think generating this file is necessary to check that we can give consistent universes to the full stdlib.